src/any_agent/callbacks/span_cost.py

```diff
@@ -20,25 +21,21 @@
 def add_cost_info(span: Span) -> None:
     """Use litellm to compute cost and add it to span attributes."""
     attributes: Mapping[str, AttributeValue] = span.attributes
-    if any(
-        key in attributes
-        for key in ["gen_ai.usage.input_tokens", "gen_ai.usage.output_tokens"]
-    ):
-        try:
-            cost_prompt, cost_completion = cost_per_token(
-                model=str(attributes.get("gen_ai.request.model", "")),
-                prompt_tokens=int(attributes.get("gen_ai.usage.input_tokens", 0)),  # type: ignore[arg-type]
-                completion_tokens=int(attributes.get("gen_ai.usage.output_tokens", 0)),  # type: ignore[arg-type]
-            )
-            span.set_attributes(
-                {
-                    "gen_ai.usage.input_cost": cost_prompt,
-                    "gen_ai.usage.output_cost": cost_completion,
-                }
-            )
-        except Exception as e:
-            msg = f"Error computing cost_per_token: {e}"
-            logger.warning(msg)
+    try:
+        cost_prompt, cost_completion = cost_per_token(
+            model=str(attributes.get(GenAI.REQUEST_MODEL, "")),
```

Comment:
Why the behavior change here? The if statement disappeared

Reply:
reverted